install/cockroach.go

```diff
@@ -387,13 +387,18 @@ tar cvf certs.tar certs
 			if c.IsLocal() {
 				cmd = `cd ${HOME}/local/1 ; `
 			}
-			cmd += `COCKROACH_CONNECT_TIMEOUT=0 `
-			cmd += cockroachNodeBinary(c, 1) + " sql --url " +
+			dir := c.Impl.NodeDir(c, nodes[i])
+			cmd += `
+if ! test -e ` + dir + `/settings-initialized ; then
+  COCKROACH_CONNECT_TIMEOUT=0 ` + cockroachNodeBinary(c, 1) + " sql --url " +
 				r.NodeURL(c, "localhost", r.NodePort(c, 1)) + " -e " +
 				fmt.Sprintf(`"
 SET CLUSTER SETTING server.remote_debugging.mode = 'any';
 SET CLUSTER SETTING cluster.organization = 'Cockroach Labs - Production Testing';
-SET CLUSTER SETTING enterprise.license = '%s';"`, license)
+SET CLUSTER SETTING enterprise.license = '%s';"`, license) + `;
```

Comment:
Do we set `set -e` when we invoke Bash? Otherwise all of these SET CLUSTER SETTINGS could fail but we’d still write the marker file.

Reply:
Good point. Switched to using `&&` instead of `;` to separate the `cockroach sql` command and the `touch` command.